mintpy/utils/readfile.py

```diff
@@ -10,7 +10,7 @@
 import os
 import re
 import warnings
-import defusedxml.ElementTree as ET
+import xml.etree.ElementTree as ET
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using xml.etree.ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called.](https://app.codacy.com/manual/insarlab/MintPy/pullRequest?prid=5671549)